src/Compilers/Core/Portable/PEWriter/Members.cs

```diff
@@ -95,6 +95,22 @@ internal static CallingConvention FromSignatureConvention(this SignatureCallingC
 
         internal static SignatureCallingConvention ToSignatureConvention(this CallingConvention convention)
             => (SignatureCallingConvention)convention & SignatureMask;
+
+        /// <summary>
+        /// Compares calling conventions, ignoring calling convention attributes.
```

Comment:
>ignoring calling convention attributes [](start = 42, length = 38)

It looks like this is done only for one input.

Reply:
Perhaps we should assert that the other one doesn't have them.

---
In reply to: [395930108](https://github.com/dotnet/roslyn/pull/42488#discussion_r395930108) [](ancestors = 395930108)